src/pandapipes/create.py

```diff
@@ -1626,6 +1696,78 @@ def create_heat_exchangers(net, from_junctions, to_junctions, diameter_m, qext_w
     return index
 
 
+def create_heat_consumers(net, from_junctions, to_junctions, diameter_m,
+                          controlled_mdot_kg_per_s=None, qext_w=None, deltat_k=None, treturn_k=None,
+                          name=None, index=None, in_service=True, type="heat_consumer", **kwargs):
+    """
+    Creates several heat consumer elements in net["heat_consumer"] from heat consumer parameters.
+
+    :param net: The net for which this heat consumer should be created
+    :type net:
+    :param from_junctions: IDs of the junctions on one side which the heat consumers will be \
+        connected with
+    :type from_junctions: Iterable(int)
+    :param to_junctions: IDs of the junctions on the other side which the heat consumers will be \
+        connected with
+    :type to_junctions: Iterable(int)
+    :param diameter_m: The heat consumers' inner diameter in [m] - only for result calculation
+    :type diameter_m: Iterable(float) or float
+    :param controlled_mdot_kg_per_s: Mass flow set point in [kg/s].
+    :type controlled_mdot_kg_per_s: Iterable(float) or float, default None
+    :param qext_w: External heat flux in [W]. If positive, heat is extracted from the network. If \
+        negative, heat is being fed into the network from a heat source.
+    :type qext_w: Iterable(float) or float, default None
+    :param deltat_k: Temperature difference set point between flow and return in [K].
+    :type deltat_k: Iterable(float) or float, default None
+    :param treturn_k: Return temperature set point in [K].
+    :type treturn_k: Iterable(float) or float, default None
+    :param name: Names of the heat consumer elements
+    :type name: Iterable(str) or str, default None
+    :param index: Force specified IDs if they are available. If None, the index one higher than the\
+            highest already existing index is selected and counted onwards for the amount of heat \
+            consumers created.
+    :type index: Iterable(int) or int, default None
+    :param in_service: True for heat consumers that are in service, False for those out of service
+    :type in_service: Iterable(bool) or bool, default True
+    :param type: Currently not used - possibility to specify certain types of heat consumers
+    :type type: Iterable(str) or str, default "heat_consumer"
+    :param kwargs: Additional keyword arguments will be added as further columns to the \
+        net["heat_consumer"] table
+    :type kwargs: dict
+    :return: index - The unique IDs of the created heat consumers
+    :rtype: int
+
+    :Example:
+        >>> create_heat_consumers(net,from_junctions=[0, 3], to_junctions=[1, 5], diameter_m=40e-3, qext_w=2000)
```

Comment:
Should not two parameter be defined according to line 1751. So qext and maybe mdot?

Reply:
yep, fixed that, thanks.